test/configure_dbus.sh

```diff
@@ -1,20 +1,34 @@
 #!/bin/bash
 
 echo "Configuring D-bus environment"
-echo 'eval "$(dbus-launch --sh-syntax)"' > /dbus.sh
+
+echo "#!/bin/bash -ex" >> /dbus.sh
```

Comment:
@sigalsax I think this will be a good idea to change `/dbus.sh` file name into `tests_executer`
and the `test/configure_dbus.sh ` filename is also not relevant I think.
WDYT?

Reply:
> I think this will be a good idea to change /dbus.sh file name into tests_executer

why tests_executor? they all env vars saved are dbus related 

>  the test/configure_dbus.sh filename is also not relevant I think.

If anything this should be test_executor IMO because we are setting up the environment and executing the tests @mbenita-Cyberark